src/components/charts/SavedValues/SavedValuesList.tsx

```diff
@@ -228,40 +243,65 @@ export default function SavedValuesList() {
                         renderEmptyState('No matching preset values', 'We couldn’t find any matching results', true)
                     ) : (
                         <>
-                            <div className="preset-values-row fw-6 cn-7 fs-12 border-bottom text-uppercase pt-8 pr-16 pb-8 pl-16">
-                                <div className="pr-16"></div>
-                                <div className="pr-16">Name</div>
-                                <div className="pr-16">Version</div>
-                                <div className="pr-16"></div>
+                            <div className="preset-values-row fw-6 cn-7 fs-12 border-bottom text-uppercase pt-8 pr-20 pb-8 pl-20">
+                                <div />
+                                <div>Name</div>
+                                <div>Version</div>
+                                <div>Last updated by</div>
+                                <div>Updated at</div>
                             </div>
-                            <div
-                                className="preset-value-list"
-                                style={{ height: 'calc(100vh - 235px)', overflowY: 'auto' }}
-                            >
+                            <div className="preset-value-list">
                                 {filteredSavedValueList.map((chartData, index) => (
```

Comment:
Aren’t we checking for the filteredSavedValueList length here?

Reply:
We're already checking for it above this statement and rendering an empty view if the length is 0.